fastly/resource_fastly_tls_subscription.go

```diff
@@ -213,6 +218,13 @@ func resourceFastlyTLSSubscriptionRead(_ context.Context, d *schema.ResourceData
 		domains = append(domains, domain.ID)
 	}
 
+	// NOTE: there must be only one certificate id included per subscription
+	// "pending" and "processing" state may not include the id (for new subscriptions)
+	var certificateId = ""
+	if len(subscription.Certificates) > 0 {
+		certificateId = subscription.Certificates[0].ID
+	}
```

Comment:
What happens if the len is zero (is that possible)? If so I presume we need to stop the logic flow.

Reply:
@Integralist As the comment suggests, it is possible we get empty `data[]` in response. But we don't need to stop the logic, we just pass it through and store `""` as the cert id with those cert state ("pending"/"processing"). We will get the cert id when the cert is actually issued (e.g., in the next TF refresh).